apps/bfd-pipeline/bfd-pipeline-idr/bfd.sql

```diff
@@ -490,7 +490,8 @@ SELECT DISTINCT
     bene_mbi.bene_mbi_obslt_dt
 FROM idr.beneficiary bene
 LEFT JOIN idr.beneficiary_mbi_id bene_mbi 
-    ON bene.bene_mbi_id = bene_mbi.bene_mbi_id;
+    ON bene.bene_mbi_id = bene_mbi.bene_mbi_id
+    where bene.idr_ltst_trans_flg = 'Y';
```

Comment:
We try to capitalize SQL operators for consistency

```suggestion
    WHERE bene.idr_ltst_trans_flg = 'Y';
```